openbas-front/src/admin/components/simulations/simulation/validation/expectations/ManualExpectations.tsx

```diff
@@ -113,11 +99,23 @@ const ManualExpectations: FunctionComponent<Props> = ({
   const validatedCount = expectations.filter(v => !R.isEmpty(v.inject_expectation_results)).length;
   const isAllValidated = validatedCount === expectations.length;
 
-  const label = isAllValidated
-    ? `${t('Validated')} (${parentExpectation.inject_expectation_score})`
-    : t('Pending validation');
+  let label;
+  let style;
+  if (!isAllValidated || !parentExpectation) {
+    label = t('Pending validation');
+    style = colorStyles.orange;
+  } else {
+    const results = parentExpectation.inject_expectation_results ?? [];
+    const hasFailed = results.some(result => result?.result === 'Failed');
```

Comment:
question: 'Failed' value is a constant ? Or can it be translated in the user's language somehow? If so, this condition would not work anymore

Reply:
It's a constant from InjectExpectationService

Reply:
I will create a pain point to handle translation later if it's ok for you.